packages/slice-machine/lib/io/Types.ts

```diff
@@ -36,11 +41,30 @@ const generateAndWriteTypes = (env: BackendEnvironment) => {
   );
 };
 
+export function getFromPackage(
+  key: "dependencies" | "devDependencies",
+  packageJson: JsonPackage | null
+): Record<string, string> {
+  if (
+    packageJson !== null &&
+    isObject(packageJson) &&
+    has(key, packageJson) &&
```

Comment:
key in packageJson would also have worked in pure typescript :)

Reply:
I tried that one a few times with no luck :/